rudder-agent/SPECS/rudder-agent.spec

```diff
@@ -44,6 +44,9 @@
 # We need to build curl since we embed openssl
 %define use_system_curl false
 
+# Same goes for the use of the local jq install vs. a bundled one
+%define use_system_jq true
```

Comment:
You must also declare jq as a dependency here when system_jq is true
And declafe jq-devel as a build-dependency too

Reply:
Not sure I fixed it properly,  but "fix coming"